src/textual/containers.py

```diff
@@ -14,45 +14,93 @@ class Container(Widget):
 
 
 class Vertical(Widget):
-    """A container widget which aligns children vertically."""
+    """A container which lays children vertically."""
```

Comment:
I don't think "lays" is quite the right way to phrase this here; `aligns` seemed to make more says, or "lays out" would work.

Reply:
I don't like "align" because that can be confused with the `align-*` styles. The children are not aligned vertically (e.g., one might be to the left and the other might be to the right).

What if I write "A container which composes children vertically."?
I don't love this wording either, but I think it is better than using "align".

Reply:
I don't think `composes` works, as it might confuse with our compose method.

How about `arranges`? "A container which arranges children vertically."

Reply:
I like "arranges".